charts/harvester-network-controller/values.yaml

```diff
@@ -60,3 +60,18 @@ helper:
   image:
     repository: rancher/harvester-network-helper
     tag: "master-head"
+
+webhook:
+  replicas: 2
```

Comment:
probably 1 is enough as don't want to increase the default system resource too much in the newer versions.